### PR TITLE
Enable helloworld testcase for examples automation.

### PR DESCRIPTION
**Pull request by craigyam**
The helloworld tests are only run for examples automation
(make test.examples).  Re-enabling them to run for examples
using the k8s native routing rules.  The integration tests for
k8s will still be disabled until the bookinfo tests are updated.

### CODE REVIEW

File: testing/test-scripts/helloworld-v1-v2-route-rules.yaml
Comment (elevran) [inline testing/test-scripts/helloworld-v1-v2-route-rules.yaml]: Don't understand why additional tag needed here.
Wanting to have v2 with 25% probability. Needs priority fix as in #512.

File: testing/test-scripts/helloworld.sh
Comment (elevran) [inline testing/test-scripts/helloworld.sh]: At least currently we can create the json and yaml rules from the same source more or less. Something to consider so we don't have duplicate code?

File: testing/test-scripts/helloworld.sh
Comment (elevran) [inline testing/test-scripts/helloworld.sh:85]: Envoy still takes sometime to restart. Don't we need the sleep?

File: testing/test-scripts/helloworld.sh
Comment (elevran) [reply testing/test-scripts/helloworld.sh]: Also, seems like controlplane and demo topologies are copied to examples using a `docker` or `k8s` prefix, but rules are copied as is. In case there is ever a divergence, we should consider adding a prefix to the rules on copying as well.

**Review by elevran — COMMENTED** [review]
Overall looks ok. Some clarifications needed per comments.

File: testing/test-scripts/helloworld-v1-v2-route-rules.yaml
Comment (craigyam) [reply testing/test-scripts/helloworld-v1-v2-route-rules.yaml]: The script clears out the previous rules before the next tests.  Without setting version=v1 in that case 3/4 of the requests get a 404.  I can update the script to keep the default routing rule and then remove that.

File: testing/test-scripts/helloworld.sh
Comment (craigyam) [reply testing/test-scripts/helloworld.sh]: The rules are currently not copied over to examples/ at all.  I'll update the automation to do that with the k8s prefix.

File: testing/test-scripts/helloworld-v1-v2-route-rules.yaml
Comment (elevran) [reply testing/test-scripts/helloworld-v1-v2-route-rules.yaml]: Or add the previous rule as well. Your call.

**Review by elevran — APPROVED** [review]
LGTM once comments are integrated into this PR

File: testing/test-scripts/helloworld-v1-v2-route-rules.yaml
Comment (craigyam) [reply testing/test-scripts/helloworld-v1-v2-route-rules.yaml]: Done

File: testing/test-scripts/helloworld.sh
Comment (craigyam) [reply testing/test-scripts/helloworld.sh]: Will investigate generating rules files for automation.  For now, copy to examples/ and prefix appropriately.

File: testing/test-scripts/helloworld.sh
Comment (craigyam) [reply testing/test-scripts/helloworld.sh:85]: Sleep was added at 76 after rules creation.

**Pull request merged**
